### PR TITLE
Adding query for all repos for an org or user

### LINKED ISSUE #6: Add polling for repos
It would be great if this could do all repos by polling GitHub for the repo list for a user/organization every time it runs.

### PR DESCRIPTION
**Pull request by jimbobbennett**
## Description

Adds polling for the repos belonging to an org, and allows them to be used instead of a list of specific repos.

## Fixes

#6 

## Type of PR
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

## Checklist
- [X] Code follows this projects style guide
- [X] I reviewed my code
- [ ] Tested in deployment environment
- [X] Tested in an alternate environment (please update here: tested locally)
- [X] Required documentation has been updated
- [ ] Other (explain: )

### CODE REVIEW

**Review by adiazulay — COMMENTED** [review]
This looks awesome! I really like that is will auto update if you make new repos.

I added a suggestion for improving the encapsulation by passing the base url in the __init__ methods. Let me know what you think.

File: README.md [hunk 1]
```diff
@@ -142,15 +142,32 @@ You can use the Data Explorer in the Azure portal to create a database and conta
...
+    1. For all repos for a user or organization, delete the declaration of the `repos` dictionary, and uncomment the lines in the `main` function that retrieve the orgs.
+
+        ```python
+        def main(mytimer: func.TimerRequest):
+            # Uncomment this to load the repos from GitHub each time
+            org = Org("<REPO-OWNER>", os.getenv("GithubApiKey"))
+            repos = org.repos()
+        ```
+
+        Replace `<USER-OR-ORG-OWNER>` with the GitHub user name or organization name.
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply README.md]: Fixed manually with other tweaks

File: github_tracker/traffic_tracker.py [hunk 2]
```diff
@@ -70,7 +106,6 @@ class Repo:
         Returns a dict of the views, clones, watchers, stars, forks, and pull
         requests for the repository
     '''
-    _API_URL_BASE = "https://api.github.com"
     _V4 = "/graphql"
 
     def __init__(self, owner, name, url, api_token):
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply github_tracker/traffic_tracker.py:111]: I'm not sure this really adds much - the `API_URL_BASE` won't ever change unless there is a new API which would need bigger changes, so why pass it in?

As a suggestion, I've renamed it to _API_URL_BASE to indicate that it's private.

File: github_tracker/__init__.py [hunk 3]
```diff
@@ -14,6 +14,10 @@
 
 
 def main(mytimer: func.TimerRequest):
+    # Uncomment this to load the repos from GitHub each time
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply github_tracker/__init__.py]: Moved

File: github_tracker/traffic_tracker.py [hunk 4]
```diff
@@ -46,6 +47,41 @@ def upload(self, data):
         '''
         self._container_client.upsert_item(data)
 
+class Org:
```
[the remark this answers is not in the document]
Comment (jimbobbennett) [reply github_tracker/traffic_tracker.py]: Renamed to UserOrOrg

**Comment by jimbobbennett** [comment]
Fixed a few suggestions. Not sure the encapsulation adds anything except potential for confusion around what should be passed in. This value will never change, so no reason to have it set manually instead of using a constant. I've added an _ to the start to indicate it's private.

**Comment by adiazulay** [comment]
Looks good, tested this on my device and everything works.

**Pull request merged**